docs/index.adoc

```diff
@@ -150,31 +150,53 @@ It *must be unique* for your application.
 
 NOTE: See <<Feature Permissions>> for details on the various states a feature can have.
 
-=== Split targeting strategies and targeting rules
+=== Strategy Splits and general targeting rules
```

Comment:
Any reason you renamed it? We call it "split targeting rules" in the UI. Where did "splits" come from?